app/components/channel_icon/index.tsx

```diff
@@ -186,16 +214,14 @@ const ChannelIcon = ({
         icon = (
             <DmAvatar
                 channelName={name}
-                isInfo={isInfo}
+                isOnCenterBg={isOnCenterBg}
+                style={commonStyles}
+                size={size}
             />
         );
     }
 
-    return (
-        <View style={[styles.container, {width: size, height: size}, style, mutedStyle]}>
-            {icon}
-        </View>
-    );
+    return (<>{icon}</>);
```

Comment:
does it need the fragment?

```suggestion
    return icon;
```

Reply:
It was complaining because `icon` can be undefined. If we set icon to `null` instead, typescript doesn't yell.